doc/QUICKSTART.md

```diff
@@ -0,0 +1,30 @@
+## QuickStart
+
+- Install yeoman and the generator: `yarn global add yo @thetribe/generator-project`
+- Create a new directory for your project and go to it
+
+### Guided mode
+
+Run `yo @thetribe/project` and follow the prompts to generate your project
+
+### Manual mode
+
+Run `yo @thetribe/project:root` to generate a bare root project and then
+[add more components to your project](#add-more-components)
```

Comment:
This probably needs a section to explain what's the difference between manual mode and guided mode or else we get the impression that it's the next step in the usage process.

Does the manual mode even something that should go in a Quickstart guide since it's mostly for an advanced usage?

Reply:
Ok I moved the advanced usage in the readme (for reference purpose - we can create a dedicated section later on, on FMP), allowing me to simplify the quickstart section (and removing the guided mode term)